HGustavs/LenaSYS
# Added titles in dropdown menu on forward- or backward button press

Testing:

1. Go to course webbprogrammering
2. Enter code example JavaScript Example 1
3. Press down the forward button
4. Click the title item in the dropdown (you should not be navigated to a new page)

https://user-images.githubusercontent.com/81631818/168545823-78b47a96-a250-4911-95f3-fb49f0b993ef.mp4

5. Press down the backward button (you should not be navigated to a new page)
6. Click the title item in the dropdown (you should not be navigated to a new page)

https://user-images.githubusercontent.com/81631818/168545802-cda6daf3-1d4a-4f83-8a40-73d4c2dcdb90.mp4
